projects/ion/src/lib/no-data/no-data.component.scss

```diff
@@ -20,4 +20,8 @@
     line-height: ion-theme(no-data-text-line-height);
     color: ion-theme(no-data-text-color);
   }
+
+  ::ng-deep ion-icon svg {
+    fill: ion-theme(no-data-icon-color);
+  }
```

Comment:
The `no-data-icon-color` does not exists
 
![image](https://github.com/user-attachments/assets/ef318afb-8bb3-42db-af4d-6f68ee56879b)

Reply:
It's already being added in #1170 , so I don't think it will be necessary to add it in this pr